app/_how-tos/use-access-controls-for-mcp-tools.md

```diff
@@ -0,0 +1,335 @@
+- `admin` - full access
+- `developer` - limited access
+- `suspended` - blocked from MCP tools
+
+{% entity_examples %}
+entities:
+  consumer_groups:
+    - name: admin
+    - name: developer
+    - name: suspended
+{% endentity_examples %}
+
+## Create Consumers
+
+Let's configure individual Consumers and assign them to groups. Each Consumer will use a unique API key and inherits group permissions which will govern access to MCP tools:
+
+{% entity_examples %}
+entities:
+  consumers:
+    - username: alice
+      groups:
+        - name: admin
+      keyauth_credentials:
+        - key: alice-key
+
+    - username: bob
+      groups:
+        - name: developer
+      keyauth_credentials:
+        - key: bob-key
+
+    - username: carol
+      groups:
+        - name: suspended
+      keyauth_credentials:
+        - key: carol-key
+
+    - username: eason
+      keyauth_credentials:
+        - key: eason-key
+{% endentity_examples %}
+
+## Configure the AI MCP Proxy plugin
+
+Now, let's configure the AI MCP Proxy plugin to apply tool-level access rules. The plugin controls which users or AI agents can see or call each MCP tool. Access is determined by Consumer groups and individual Consumers using allow and deny lists. A tool ACL replaces the default rule when present.
+
+The table below shows the effective permissions for the configuration:
+
+<!-- vale off -->
+{% table %}
+columns:
+  - title: MCP Tool
+    key: tool
+  - title: Admin group
+    key: admin
+  - title: Developer group
+    key: developer
+  - title: Eason consumer
+    key: eason
+  - title: Suspended group
+    key: suspended
+
+rows:
+  - tool: "`list_users`"
+    admin: Yes
+    developer: No
+    eason: Yes
+    suspended: No
+  - tool: "`get_user`"
+    admin: Yes
+    developer: Yes
+    eason: No
+    suspended: No
+  - tool: "`list_orders`"
+    admin: Yes
+    developer: Yes
+    eason: No
+    suspended: No
+  - tool: "`list_orders_for_user`"
+    admin: Yes
+    developer: Yes
+    eason: No
+    suspended: No
+  - tool: "`search_orders`"
+    admin: Yes
+    developer: No
+    eason: No
+    suspended: No
+{% endtable %}
+<!-- vale on -->
+
+The following plugin configuration applies the ACL rules for the MCP tools shown in the table above:
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: ai-mcp-proxy
+      route: mcp-acl-route
+      config:
+        mode: passthrough-listener
+        include_consumer_groups: true
+        default_acl:
+          - scope: tools
+            allow:
+              - "developer"
+              - "admin"
+            deny:
+              - "suspended"
+        logging:
+          log_payloads: false
+          log_statistics: true
+          log_audits: true
+        tools:
+          - description: List users
+            name: list_users
+            acl:
+              allow: ["admin", "eason"]
+              deny: ["developer"]
+          - description: Get user
```

Comment:
Maybe we can use same yaml list style as other files. Use nested format maybe better